doc/source/api.rst

```diff
@@ -246,6 +246,24 @@ Testing
 
    test
 
+.. currentmodule:: pandas.plotting
+
+Plotting
+--------
+
+Miscellaneous
+~~~~~~~~~~~~~
+
```

Comment:
Can you add a small sentence saying that those functions are contained in the `pandas.plotting` module?

Reply:
I'll add this too.